### ISSUE TITLE
Emmet not removing custom snippets when snippets path changed 

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode#108812]
Version: 1.51.0-insider (user setup)
Commit: a522f37e37735c0cbe7244680250ef73d9054dd1
Date: 2020-10-15T06:39:34.462Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042

Steps to Reproduce:

1. Add an Emmet snippets file to the project along with a test HTML file https://code.visualstudio.com/docs/editor/emmet#_using-custom-emmet-snippets
2. Change `emmet.extensionPath` setting to point to new snippets file and save the settings file. Custom snippets should now show up in test HTML file.
3. Change `emmet.extensionPath` setting to point to directory without snippets file and save the settings file. 
**Expected: custom snippets should stop showing up in test HTML file. 
Actual: custom snippets still show up in test HTML file.**

Does this issue occur when all extensions are disabled?: N/A

**Comment by rzhao271** [comment]
\closedWith fbc4c87

**Comment by rzhao271** [comment]
For verifiers:

1. Follow the steps for #104962 (comment)
2. Verify a snippet works. Type it and when it shows up in the intellisense, select it to expand it.
3. Change emmet.extensionsPath to point to any path without a snippet file.
4. Confirm the snippets do not show up after saving the settings.json file.

**Comment by aeschli** [comment]
I still see the emmet snippet after changing the setting to a path that doesn't exist.

![image](https://user-images.githubusercontent.com/6461412/97443727-a6f08200-192b-11eb-85e3-3015e7bce1d2.png)

Version: 1.51.0-insider
Commit: e5c0007ce0e9a3816679cba674d03376c03726e3
Date: 2020-10-27T05:33:58.966Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.3.0-64-generic

**Comment by rzhao271** [comment]
\closedWith e94a1bd7effc5ed441901ed7846193372b4b752f

**Comment by meganrogge** [comment]
@rzhao271 It could be that the change you made an hour ago hasn't yet made it into the build, but when I set the path to be an empty string, it's still using the custom snippet.

![recording (15)](https://user-images.githubusercontent.com/29464607/97480044-23756780-1910-11eb-8f8f-92f17d2af1e4.gif)

**Comment by rzhao271** [comment]
@meganrogge Cannot reproduce. The setting is `emmet.extensionsPath` with an s, but even when I change the setting to `emmet.extensionPath: ""` the snippets disappear for me, even in the current insiders build.

I wonder if in your case there's another settings file in the settings editor that's overriding that local settings.json file.

**Comment by meganrogge** [comment]
I don't see any additional settings.json files in my project and am unable to get the snippets to disappear regardless of how I change the path @rzhao271

**Comment by rzhao271** [comment]
Is this how you accessed the settings file?
Or is there a way to use a custom settings file that I wasn't sure about @meganrogge

![recording (5)](https://user-images.githubusercontent.com/7199958/97595056-8de5e080-19c0-11eb-8ca8-e194f25fd0e1.gif)

**Comment by meganrogge** [comment]
@rzhao271 it works for me now and I'm not sure why. but nice work!

**Comment by rebornix** [comment]
I'm not able to see emmet snippets with setting `    "emmet.extensionsPath": "/Users/penlv/code/vscode/.vscode/snippets.json",
`, even after reloading the workspace. Would love to know how @meganrogge or @rzhao271 get it to work.

**Comment by rzhao271** [comment]
@rebornix I did open another issue upon more testing.

However, there's two potential things that could be preventing you from seeing the snippets still:

1. Your snippets file is not in the following format https://code.visualstudio.com/docs/editor/emmet#_using-custom-emmet-snippets
2. You tried typing the snippets immediately. There's a delay time between opening up an HTML/CSS file and typing the snippets.

**Comment by meganrogge** [comment]
@rebornix I had to use // instead of / in the path to my snippets.json file. make sure your path is full/correct. it was always working for me which was the problem. similar to aeschli, I was able to put an invalid path in the settings file and it would still show the snippets. When I most recently tried, it worked and I think that's because of Raymond's [fix ](https://github.com/microsoft/vscode/commit/e94a1bd7effc5ed441901ed7846193372b4b752f) which hadn't yet been released when I tried it.
